subset/dhcp/README.md

```diff
@@ -15,11 +15,4 @@ Due to this, the DHCP module requires the ipaddr module to run successfully:
 - Reconnect device and check for DHCP request. 
 #### Result Cases:
 - PASS: A DHCP request has been received by the device after the port has been disconnected and connected.
-- FAIL: No DHCP request has been received.
-
-## Integrated Testing
-
-To provide full test coverage for this module, a static IP is configured for the faux-2 interface.
-By configuring a static IP and disabling dhclient no DHCP requests are sent on this device.
-
-The IP used is <strong>10.20.0.7</strong>.
+- FAIL: No DHCP request has been received (because the device has a static IP set or it is not configured to send a DHCP request on reconnect).
```

Comment:
sorry to be difficult, but this isn't quite right.  I think you need:

`FAIL: No DHCP request was received (this will also be the case if the target is using a static IP).`

We don't know if it's because of "configuration" or not.